recommendations.md

```diff
@@ -61,6 +61,13 @@ On the catalog, this means that you should, where possible:
         privileged: true
   ```
 
+## Be as portable and compatible as possible
+
+Make use of recent Kubernetes and Tekton features only when a user
+will expect it from the task's purpose. Your task may be of great use
+to users that have good reason not to upgrade right now. Your task
+should include the `tekton.dev/pipelines.minVersion`.
```

Comment:
nit: `should` -> `must`.  The field is mandatory according to the PR template.

Reply:
nice catch!